docs/diagnostics/PX1007.md

```diff
@@ -29,7 +32,22 @@ The diagnostic is not displayed for the following items:
  - Members of public entities (except nested public types and DAC property fields).
  - All members of a public entity if the entity has the `exclude` XML tag or is marked with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
  - Class fields of DACs. 
- - System fields of DACs, which are the following: `CreatedByID`, `CreatedByScreenID`, `CreatedDateTime`, `LastModifiedByID`, `LastModifiedByScreenID`, `LastModifiedDateTime`, `TStamp`, and `NoteID`.
+ - Well-known DAC fields, which are the following:
+   - `Selected`
+   - `Attributes`
+   - `GroupMask`
+ - System fields of DACs, which are the following: 
+   - `CompanyID`
+   - `CompanyMask`
+   - `DeletedDatabaseRecord`
+   - `CreatedByID`
+   - `CreatedByScreenID`
+   - `CreatedDateTime`
+   - `LastModifiedByID`
+   - `LastModifiedByScreenID`
+   - `LastModifiedDateTime`
+   - `TStamp`
+   - `NoteID`.
 
 If you suppress this diagnostic with a comment, the default comment message is the following: _Acuminator disable once PX1007 NoXmlCommentForPublicEntityOrDacProperty to be documented later_.
```

Comment:
The default justification was deleted. Developers shouldn't suppress the diagnostic to exclude API from the documentation. They should use exclude tag for this.